pjlib/src/pj/os_info.c

```diff
@@ -70,26 +70,26 @@
 #endif
 
 
-static char *ver_info(pj_uint32_t ver, char *buf)
+static char *ver_info(pj_uint32_t ver, char *buf, unsigned buf_size)
 {
-    pj_size_t len;
-
     if (ver == 0) {
         *buf = '\0';
         return buf;
     }
 
-    sprintf(buf, "-%u.%u",
+    pj_ansi_snprintf(buf, buf_size, "-%u.%u",
             (ver & 0xFF000000) >> 24,
             (ver & 0x00FF0000) >> 16);
```

Comment:
a code indent problem

Reply:
intentional to minimize code change (re: git blame)